test/lib/usd/translators/UsdImportMeshTest/Mesh.usda

```diff
@@ -31,18 +31,69 @@ def Xform "World"
         int[] primvars:st:indices = [0, 1, 2, 3, 3, 2, 4, 5, 5, 4, 6, 7, 7, 6, 8, 9, 1, 10, 11, 2, 12, 0, 3, 13]
     }
 
+    def Mesh "LeftHandedSubdivMesh" (
+        kind = "component"
+    )
+    {
+        uniform token orientation = "leftHanded"
+        float3[] extent = [(0.5, 0.5, 0.5), (1.5, 1.5, 1.5)]
+        token faceVaryingLinearInterpolation = "all"
+        int[] faceVertexCounts = [4, 4, 4, 4, 4, 4]
+        int[] faceVertexIndices = [0, 1, 3, 2, 2, 3, 5, 4, 4, 5, 7, 6, 6, 7, 1, 0, 1, 7, 5, 3, 6, 0, 2, 4]
+        token interpolateBoundary = "edgeOnly"
+        point3f[] points = [(0.5, 0.5, 1.5), (1.5, 0.5, 1.5), (0.5, 1.5, 1.5), (1.5, 1.5, 1.5), (0.5, 1.5, 0.5), (1.5, 1.5, 0.5), (0.5, 0.5, 0.5), (1.5, 0.5, 0.5)]
+        color3f[] primvars:displayColor = [(0.13320851, 0.13320851, 0.13320851)] (
+            customData = {
+                dictionary Maya = {
+                    bool generated = 1
+                }
+            }
+        )
+        texCoord2f[] primvars:st = [(0.375, 0), (0.625, 0), (0.625, 0.25), (0.375, 0.25), (0.625, 0.5), (0.375, 0.5), (0.625, 0.75), (0.375, 0.75), (0.625, 1), (0.375, 1), (0.875, 0), (0.875, 0.25), (0.125, 0), (0.125, 0.25)] (
+            interpolation = "faceVarying"
+        )
+        int[] primvars:st:indices = [0, 1, 2, 3, 3, 2, 4, 5, 5, 4, 6, 7, 7, 6, 8, 9, 1, 10, 11, 2, 12, 0, 3, 13]
+    }
+
     def Mesh "PolyMesh" (
         kind = "component"
     )
     {
-        float3[] extent = [(-0.5, -0.5, -0.5), (0.5, 0.5, 0.5)]
```

Comment:
I changed coordinates so that the meshes and subdiv don't overload, so it can more easily be manually viewed in Maya or others.